test/cppsuite/tests/search_near_01.cpp

```diff
@@ -167,7 +167,7 @@ class search_near_01 : public test {
                     for (uint64_t k = 0; k < ALPHABET.size(); ++k) {
                         std::string key = {ALPHABET.at(i), ALPHABET.at(j), ALPHABET.at(k)};
                         evict_cursor->set_key(evict_cursor.get(), key.c_str());
-                        evict_cursor->search_near(evict_cursor.get(), &cmpp);
```

Comment:
np: We technically don't need the evict cursor anymore in this test

Reply:
As discussed, if we want to make changes to this test, it would be better in another ticket but I agree with your observation.